### PR TITLE
CLOUDP-153400: Remove kubectl

### PR DESCRIPTION
**Pull request by Sugar-pack**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

**Review by igor-karpukhin — APPROVED** [review]
Great job 👍

File: test/e2e/private_link_test.go [hunk 1]
```diff
@@ -34,7 +32,6 @@ var _ = Describe("UserLogin", Label("privatelink"), func() {
 	var testData *model.TestDataProvider
 
 	_ = BeforeEach(func() {
-		Eventually(kubecli.GetVersionOutput()).Should(Say(K8sVersion))
```
Comment (fabritsius) [inline test/e2e/private_link_test.go:37]: I'm surprised that removing this doesn't cause any problems :)
I thought this is like "tell when you're ready flag".
Great if we don't need it at all 👌

**Review by fabritsius — APPROVED** [review]
Looks great 👍 
Just have one little question :)

File: test/e2e/private_link_test.go (hunk 1 above)
Comment (Sugar-pack) [reply test/e2e/private_link_test.go:37]: Yeah. Let's add it to the workflow file

Comment (Sugar-pack) [reply test/e2e/private_link_test.go:37]: It's just a compression between the existing and expected versions of kubectl

**Review by helderjs — APPROVED** [review]

**Pull request merged**